extensions/dockerfiles/deployment/src/main/java/io/quarkus/dockerfiles/deployment/DockerfilesConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.dockerfiles.deployment;
+
+import java.util.Optional;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot
+public class DockerfilesConfiguration {
+
+    static final String DEFAULT_JVM_FROM = "registry.access.redhat.com/ubi8/openjdk-21:1.19";
+    static final String DEFAULT_NATIVE_FROM = "registry.access.redhat.com/ubi8/ubi-minimal:8.9";
+
+    /**
+     * The from image to use for JVM based Dockerfiles
+     */
+    @ConfigItem(defaultValue = "registry.access.redhat.com/ubi8/openjdk-21:1.19")
+    String jvmFrom;
+
+    /**
+     * The from image to use for native based Dockerfiles
+     */
+    @ConfigItem(defaultValue = "registry.access.redhat.com/ubi8/ubi-minimal:8.9")
+    String nativeFrom;
```

Comment:
Not for this PR but we need to centralize all these images reference somewhere. We have them spread all over our code and it's a recipe for disaster.

I will create an issue about this so that we discuss it when @cescoffier is back.